### DISCUSSION TITLE
next v13.2.4 component scss style does not work in _document.js

### DISCUSSION THREAD

**Discussion by Lerbron** [vercel/next.js#47136] [Help]
### Summary

After I introduced xx.module.scss in a component, classname was generated, but style did not work。
![截屏2023-03-15 09 52 20](https://user-images.githubusercontent.com/23184874/225188592-7d14b3c5-bd81-4293-9740-1d7ff6f99dbf.png)
![截屏2023-03-15 10 25 55](https://user-images.githubusercontent.com/23184874/225188877-8e523c0f-007e-4a70-a184-22355e49dabe.png)
![截屏2023-03-15 11 03 29](https://user-images.githubusercontent.com/23184874/225194705-aa8f9ee9-16b5-4ae8-bd01-8e7389f5f0e8.png)
![截屏2023-03-15 11 03 48](https://user-images.githubusercontent.com/23184874/225194713-d07eba5f-26b4-4546-b0fa-2ffb83780180.png)

### Additional information

```js
Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 22.3.0: Mon Jan 30 20:42:11 PST 2023; root:xnu-8792.81.3~2/RELEASE_X86_64
    Binaries:
      Node: 18.10.0
      npm: 8.19.2
      Yarn: 1.22.4
      pnpm: N/A
    Relevant packages:
      next: 13.2.4
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0
```

**Comment by icyJoseph** [comment]
Is JavaScript enabled on the browser?

**Reply by Lerbron** [reply]
Of course.When i use Layout in _app.js does work.but in _document.js does not work.

**Reply by icyJoseph** [reply]
oh my bad. I did not read the _document bit... had just been bitten by JS being disabled on the browser (during dev mode).

I think you have to refer to this comment: https://github.com/vercel/next.js/issues/3714#issuecomment-363750859

**Reply by Lerbron** [reply]
Thanks
